understanding/21/non-text-contrast.html

```diff
@@ -83,8 +83,82 @@ <h4 id="related-scs">Relationship with Use of Color and Focus Visible</h4>
 
 				<p>Using a change of contrast for focus and other states is a technique to differentiate the states. This is the basis for <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G195" class="general">G195: Using an author-supplied, highly visible focus indicator</a>, and more techniques are being added.</p>
 
+
+				<h4 id="related-focus">Relationship with Focus Visible</h4>
+				
 				<p>In combination with <a href="https://www.w3.org/TR/WCAG21/#focus-visible">2.4.7 Focus Visible</a>, the visual focus indicator for a component <em>must</em> have sufficient contrast against the adjacent background when the component is focused, except where the appearance of the component is determined by the user agent and not modified by the author.</p>
-        <p>Note that this Success Criterion does not directly compare the focused and unfocused states of a control - if the focus state relies on a change of color (e.g., changing <em>only</em> the background color of a button), this Success Criterion does not define any requirement for the difference in contrast between the two states.</p>
+
+				<p>Most focus indicators appear outside the component, in that case it needs to contrast with the background that the component is on. Other cases include focus indicators which are:</p>
```

Comment:
that comma looks awkward. perhaps better `Most focus indicators appear outside of the component. In that case, it needs to contrast...`

Reply:
or, alternatively, a hyphen `outside of the component – in that case ...`